main/views.py

```
@@ -34,11 +36,52 @@ def add_employee(request):
-    pass
-def view_xml_by_id(request, id):
-    pass
-def view_json_by_id(request, id):
-    pass
\ No newline at end of file
+    products = Product.objects.all()
+    json_data = serializers.serialize("json", products)
+    return HttpResponse(json_data, content_type="application/json")
+
+
+def view_xml_by_id(request, pk):
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)